tests/manual-test-cases/Group19-VIC-Product/19-5-Configuration-1-VC6.5-ESX6.5-VIC-OVA-Harbor-Insecured.md

```diff
@@ -0,0 +1,99 @@
+Test 19-5 - Configuration 1 VC6.5 ESX6.5 VIC OVA Insecured
+=======
+
+#Purpose:
+To verify the VIC Product (Engine/Harbor) work using insecure registry with a VC6.5 and ESX6.5 server
+
+#References:
+[1 - VIC+Harbor Integration Test Plan](https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=corevc&title=VIC+-+Harbor+Integration+Test+Plan)
+
+#Environment:
+* This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
+* Login to Harbor using an LDAP user1 (say admin role)
+* Create a Project named say 'vic-harbor' (publicity off)
+* Login with 2 other LDAP users (say user2, user3) and logout - Limitation (Issue#)
+* Login as user1 and add other users with different roles (developer, guest) under Project 'vic-harbor'
+* Prepare 3 windows client machines (3rd client machine could be a Linux machine as we couldn't figure out the VT-X issue for running docker on Windows 7/10 VM)
```

Comment:
Also, why are 3 client machines required? I don't see anything in the `.robot` using different clients